src/Nethermind/Nethermind.Core/ConsensusRequests/ConsensusRequest.cs

```diff
@@ -26,6 +27,9 @@ public class ConsensusRequest
     [JsonIgnore]
     public byte[]? PubKeyField { get; set; }
 
+    [JsonIgnore]
+    public byte[]? TargetPubkeyField { get; set; }
```

Comment:
to consider: you could reuse WithdrawalCredentialsField, of course with consider naming as well